README.md

```diff
@@ -192,6 +192,7 @@ See [remarkable.guide](https://remarkable.guide/tech/factory-reset.html) for mor
 - [rmWebUiTools](https://github.com/LinusCDE/rmWebUiTools) - View a file tree, see statistics and export/backup all files with some simple scripts utilizing the web ui.
 - [send_by_rmapi](https://github.com/LisaGlaser/send_by_rmapi) - A Calibre plugin to send books to your reMarkable, making use of rmapi.
 - [RemarkableLamyEraser](https://github.com/isaacwisdom/RemarkableLamyEraser/) - Supports Lamy Al Star stylus button to erase or undo for reMarkable tablets.
+- [rm2anki](https://github.com/stelzch/rm2anki) - Template and program to convert a reMarkable notebook into an Anki card deck.
```

Comment:
```suggestion
- [rm2anki](https://github.com/stelzch/rm2anki) - Convert a reMarkable notebooks into an Anki card deck.
```
I think this will be a better option, maybe also having an entry under templates that points at https://github.com/stelzch/rm2anki/tree/main/template for the template itself?

Reply:
Thanks, I edited it according to your suggestion.

I don't think having a separate entry for the template is necessary, because on its own the template is not as useful. However, if you insist I'd reconsider :)